python/tests/rtl/test_user_session.py

```diff
@@ -0,0 +1,125 @@
+# pylint: disable=redefined-outer-name
+
+import pytest
+
+from looker.rtl.user_session import UserSession
+from looker.rtl.api_settings import ApiSettings
```

Comment:
even for test code I prefer we'd stick with the "only import modules" convention